lib/resources/may2015/medication_statement.yaml

```diff
@@ -0,0 +1,282 @@
+---
+name: MedicationStatement
+field_name_base_url: http://hl7.org/fhir/2015May/medicationstatement-definitions.html#MedicationStatement
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  example: MedicationStatement
+  note: resourceType must be MedicationStatement
+  url: http://hl7.org/fhir/2015May/json.html#resources
+
+- name: id
+  required: 'Yes'
+  cardinality: 0..1
+  type: id
+  description: The logical id of the resource to update.
+  example: '{"id": "123412"}'
+  note: The id value must match the MedicationStatement/&lt;id> value.
+  url: http://hl7.org/fhir/2015May/resource-definitions.html#Resource.id
+  action: update
+
+- name: contained
+  required: 'Yes'
+  cardinality: 0..*
+  type: Resource
+  description: Contained, inline Resources.
+  example: |
+               {"contained": [
+                 {"resourceType": "Medication",
+                    "id": "2750067",
+                    "code": {
+                      "coding": [
+                        {"system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+                         "code": "1191",
+                         "display": "Aspirin",
+                         "userSelected": false}],
+                      "text": "aspirin"}}]}
+  url: http://hl7.org/fhir/2015May/domainresource-definitions.html#DomainResource.contained
+  note: It should contain medication resource with medication.code
+  children:
+
+  - name: code
+    required: 'No'
+    cardinality: 0..1
+    type: CodeableConcept
+    description:  Codes that identify this medication.
+    example: |
+               {"code": {
+                 "coding": [
+                   {"system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+                    "code": "1191",
+                    "display": "Aspirin",
+                    "userSelected": false}],
+                 "text": "aspirin"}}
+    url: http://hl7.org/fhir/2015May/medication-definitions.html#Medication.code
+    note: This code is only belongs where the resource is a Medication.
+    binding:
+      description: A code that defines the type of medication.
+      terminology:
+      - display: RxNorm
+        system: http://www.nlm.nih.gov/research/umls/rxnorm
+
+- name: patient
+  required: 'Yes'
+  cardinality: 0..1
```

Comment:
This is required by the Cerner FHIR server.
@cerbs, how did we handle this situation in other resources? Should this be marked as required?